Modules/Loadable/Markups/Widgets/Resources/UI/qMRMLMarkupsFiducialProjectionPropertyWidget.ui

```diff
@@ -94,7 +94,7 @@
    <item row="5" column="1">
     <widget class="QCheckBox" name="pointOutlinedBehindSlicePlaneCheckBox">
      <property name="toolTip">
-      <string>Fiducial projection is displayed filled (opacity = Projection Opacity) when on top of slice plane, outlined when behind, and with full opacity when in the plane. Outline isn't used for some glyphs (Dash2D, Cross2D, Starburst).</string>
+      <string>Markup projection is displayed filled (opacity = Projection Opacity) when on top of slice plane, outlined when behind, and with full opacity when in the plane. Outline isn't used for some glyphs (Dash2D, Cross2D, Starburst).</string>
```

Comment:
I need to check this. For a projection is the entire markup projected where say a line node is projected meaning both the control points and the line that connects the control points are projected? That’s what I wrote it as the projection of the markup rather than just the projection of the control points.

Reply:
I believe only the control points are projected. I can imagine that a full projection could be useful in certain use cases, but I don't think it is implemented.

Reply:
Confirmed that only the control points for markups are projected.
![image](https://user-images.githubusercontent.com/15837524/134905197-67b29ea7-4246-43b1-aae4-0545af011212.png)
![image](https://user-images.githubusercontent.com/15837524/134905218-168a39b9-0fbd-4f99-8369-01e38c81fafe.png)